test/hotspot/jtreg/runtime/ErrorHandling/CreateCoredumpOnCrash.java

```diff
@@ -62,7 +63,8 @@ public static void main(String[] args) throws Exception {
             runTest("-XX:-CreateMinidumpOnCrash").shouldContain("CreateCoredumpOnCrash turned off, no core file dumped")
             .shouldNotHaveExitValue(0);
         } else {
-            OutputAnalyzer oa = new OutputAnalyzer(Runtime.getRuntime().exec("ulimit -c"));
+            String exec_cmd[] = {"sh", "-c", "ulimit -c"};
+            OutputAnalyzer oa = new OutputAnalyzer(Runtime.getRuntime().exec(exec_cmd));
```

Comment:
You could use `ProcessTools.executeProcess("sh", "-c", "ulimit -c");`

For that matter there are related utilities in ./lib/jdk/test/lib/util/CoreUtils.java - but maybe not worth trying to adapt to current needs.